tests/test_affinities.py

```diff
@@ -253,6 +254,20 @@ def test_to_new_all_equal(self):
         new_p = new_p.toarray()
         np.testing.assert_allclose(new_p.sum(axis=1), np.ones(self.y.shape[0]))
 
+    def test_symmetrize_mean_has_two_distinct_values(self):
+        aff = Uniform(self.x, k_neighbors=10, symmetrize="mean")
+        values = aff.P.data
+        self.assertEqual(len(np.unique(values)), 2)
+
+    def test_symmetrize_max_has_one_distinct_values(self):
+        aff = Uniform(self.x, k_neighbors=10, symmetrize="max")
+        values = aff.P.data
+        self.assertEqual(len(np.unique(values)), 1)
+
+    def test_symmetrize_with_inavlid_parameter(self):
```

Comment:
inavlid -> invalid

Reply:
Ah, yes, thanks! These typos just sneak in there